lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -243,7 +241,7 @@ abstract class TextEditingStrategy {
   ///
   /// The position will be updated everytime Flutter Framework sends
   /// 'TextInput.setEditableSizeAndTransform' message.
-  void updateElementPosition(_GeometricInfo geometricInfo);
```

Comment:
Why are we renaming these?

In the last 6 months we renamed this specific method and it's input 3 times.

My comment applies here and everywhere in the PR, can you please explain the reasoning behind the new names?

Reply:
Because we are not only positioning the element but also sizing it. I decided that "placement" has less bias than "position", which in CSS and in Flutter only means the coordinate of the top-left corner and does not have anything to do with the size. Other possible bias-free alternatives are "layout" and "geometry", e.g. `updateElementLayout` or `updateElementGeometry`.